### DISCUSSION TITLE
Less efficient Next after upgrading from 13.2.4 => 14.0.2 on GCP

### DISCUSSION THREAD

**Discussion by AdamZajler** [vercel/next.js#58822] [Help]
### Summary

I tried to post it as issue, but without reproduction i cannot post on Issues :sweat_smile: (https://github.com/vercel/next.js/issues/58821)

### Additional information

Currently on Next 13.2.4 we have 6 instances and usage of 50% in CPU & RAM.
After upgrading to 14.0.2 we have a lot of 500 errors after entering the website, soo we pushed up to 9 instances and now we have usage around ~70% in CPU & RAM and still having few 500 erros. Edit: (CPU usage is 80% & we also pushed form 16 to 24CPU's).

This usage is normal after doubling our instances but the problem is that we have feeling that Next 14 is less efficient then previous versions?

(below usage with 6 instances)
![image](https://github.com/vercel/next.js/assets/79362673/7853cbe0-8521-4eb6-8664-4358c6d728e4)

(below usage with 9 instances)
![image](https://github.com/vercel/next.js/assets/79362673/1c0fceb6-fb8e-41e2-8e6a-806992856dca)

========================================================================================================

We don't use app route yet.

We tried setting images: { unoptimized: true } => it does not change anything.

We tried upgrading Node.js to 20.9 but it consumed more resources than 18.18.2 & it wasn't as stable as 18.18.2; so we stayed on that version.

We cannot try canary version because those issues doesn't shows on our staging environments; it could be only seen on production where movement on site is 3k+; and we can't risk site going down again.

Does anyone have similar issue?

**Comment by gregrafferty** [comment]
We have also seen a similar trend in our CPU usage on AWS ECS when we moved from v13.2.4 to v13.5.6 in our last deployment a few days ago. No noticable uptick in requests but CPU utilization essentially doubled. None of the functionality we've added recently appears to be a reason for the change either. We've doubled our CPU allocation to mitigate it.

Have you had any luck improving the situation?

**Reply by pzawadzkipz** [reply]
Yup, similar issue but running it in AWS +~10-20% of CPU usage without any significant changes to code-base

**Comment by LokeshParthiban** [comment]
We are hitting similar problem. All our pages are client side rendered, nothing much happens on the server-side. Despite that, it takes 0.3 core with just 10 tabs hitting the routes every 10 seconds. It wasn't this bad with Next v13.x.
Am using Next.js 14.2.4 with Node 20.15.1.
